hail/src/main/scala/is/hail/expr/ir/streams/EmitStream.scala

```diff
@@ -1269,15 +1267,6 @@ object EmitStream {
             override def close(cb: EmitCodeBuilder): Unit = {}
           }
 
-          val innerStreamCode = EmitCode.fromI(mb) { cb =>
-            if (childProducer.requiresMemoryManagementPerElement)
```

Comment:
this code shouldn't have been here. It's on the stream consumer to set these regions -- these were being leaked!